### ISSUE TITLE
Incorporate Accessibility Finding, Bugs, Unmet WCAG standards into Bugs & Bug Severity

### ISSUE THREAD

**Issue by bischoffa** [department-of-veterans-affairs/va-mobile-app#9090]
**Description:** 

**Notes:** 
 - Supported by Qart per convo early July 2024

**AC**

**Comment by brea11y** [comment]
@bischoffa – All of my tickets have been updated with estimates, labels, and severity levels for bugs. I believe the last step will be to determine where we should document accessibility bug severity. Should we consider updating / adding it to the [issue severity page on doc site](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA/#issue-severity)?

My thoughts are:
- **Critical (sev-1):**
   - We are in direct violation of a WCAG guideline (i.e. not having proper focus effects), and/or
   - There is an accessibility issue that is causing a major impact to Veterans with disabilities (i.e. when snackbar appears, a user can't switch the focus to it)
- **High (sev-2):**
   - An accessibility fix that could be improved even though we are following WCAG guidelines (i.e. improving the pagination experience for screen reader users), and/or
   -  A minor accessibility bug that we should fix, but isn't causing a major impact to a Veteran's experience (i.e. a line of copy should be a heading, but isn't currently labeled as a heading or a button being improperly labeled as a link)

**Comment by bischoffa** [comment]
Met with Brea on 8/8. Agreed with her recommendation listed above. Suggested chatting with Therese to review this recommendation and also best place to update this. 

Also recommended to see how this will impact bug tickets in order to determine the best way to roll this out. Example if this introduces a lot of Sev-2 bugs then maybe we need to see how we rollout this change vs a hard cut off. Tim may have some thoughts to.  

Also recommend identifying what a Sev-1 Accessibility bug looks like as it will also help rationalize the decisions for Sev-2 and Sev-3. Example App no longer supports light mode, Screenreader is not working for all / most OS app users. 

Overall I think this is a great thing to implement just need to determine how to implement.

**Comment by brea11y** [comment]
Will review this with @TKDickson in our 1:1 and decide the best place to document this. I will create a new ticket to implement the updates based on that conversation.
